doc/classes/AnimatedSprite2D.xml

```diff
@@ -5,34 +5,63 @@
+				See also [method stop].
+			</description>
+		</method>
 		<method name="play">
 			<return type="void" />
-			<param index="0" name="anim" type="StringName" default="&amp;&quot;&quot;" />
-			<param index="1" name="backwards" type="bool" default="false" />
+			<param index="0" name="name" type="StringName" default="false" />
+			<param index="1" name="custom_speed" type="float" default="1.0" />
+			<param index="2" name="from_end" type="bool" default="false" />
+			<description>
+				Plays the animation with key [param name]. If [param custom_speed] is negative and [param from_end] is [code]true[/code], the animation will play backwards (which is equivalent to calling [method play_backwards]).
+				If this method is called with that same animation [param name], or with no [param name] parameter, the assigned animation will resume playing if it was paused, or restart if it was stopped (see [method stop] for both pause and stop). If the animation was already playing, it will keep playing.
+			</description>
+		</method>
+		<method name="play_backwards">
+			<return type="void" />
+			<param index="0" name="name" type="StringName" default="&amp;&quot;&quot;" />
 			<description>
-				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [param backwards] is [code]true[/code], the animation is played in reverse.
-				[b]Note:[/b] If [member speed_scale] is negative, the animation direction specified by [param backwards] will be inverted.
+				Plays the animation with key [param name] in reverse.
+				This method is a shorthand for [method play] with [code]custom_speed = -1.0[/code] and [code]from_end = true[/code], so see its description for more information.
 			</description>
 		</method>
 		<method name="stop">
 			<return type="void" />
 			<description>
-				Stops the current [member animation] at the current [member frame].
-				[b]Note:[/b] This method resets the current frame's elapsed time and removes the [code]backwards[/code] flag from the current [member animation] (if it was previously set by [method play]). If this behavior is undesired, set [member playing] to [code]false[/code] instead.
+				Stops the currently playing animation. The animation position is reset to [code]0[/code] and the playback speed is reset to [code]1.0[/code]. See also [method pause].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="animation" type="StringName" setter="set_animation" getter="get_animation" default="&amp;&quot;default&quot;">
-			The current animation from the [member frames] resource. If this value changes, the [code]frame[/code] counter is reset.
+			The key of the current shown animation.
+		</member>
+		<member name="autoplay" type="String" setter="set_autoplay" getter="get_autoplay" default="&quot;&quot;">
```

Comment:
I love it, I've wanted to suggest `autoplay` for a long time. But does it really have to be a `string` and not a `bool` (autoplay the current animation)? I know that in `AnimationPlayer` you can select an animation to autoplay, but I think this is due to the interface of the editor, and not a need.

Reply:
As you can see in the video, this fix makes the current animation easily toggled by simply selecting the animation name in SpriteEditor. For example, it is possible to change the current animation to check a particular animation and then forget to change it back to its original state.

I have a little concern about whether that would be a good implementation, and could be addressed to some extent by separating the assigned_animation and current_animation, as AnimationPlayer does, but I think that would be too expensive for AnimatedSprite.

Also, considering that AnimatedSprite may be played by AnimationPlayer instead of AnimatedSprite's play(), it would be better to simply have only one animation property.

As a result, it is better to have a String to prevent to change the current_animation inadvertently and autoplay an unpredictable animation.

Reply:
Ok, I completely agree with you. I messed with `AudioStreamPlayer.autoplay` and `Timer.autostart` (it's probably too late to rename it now) which are `bool`.